src/PagingControl/PagingControl.js

```diff
@@ -256,6 +255,7 @@ module.exports = kind(
 			return;
```

Comment:
That crossed my mind, but I decided to preserve it as it was pre-existing. We would still need the call to `endHold` in `spotlightBlurred`, regardless. I guess the only case where this would break something would be if someone had subkinded `PagingControl` and had come to rely on the parameters sent to their override of `endHold`.